apps/services/search-indexer/config/template-en.json

```diff
@@ -23,6 +23,9 @@
           },
           "stemmed": {
             "type": "text"
+          },
+          "keyword": {
+            "type": "keyword"
```

Comment:
You mean the `sort` field right?
I had a problem with that because Kibana doesn't like that in the icelandic index the sort type is `icu_collation_keyword` but in the english index it is `keyword`. I am not sure what you are using sort for so I created a new one that had consistent types between the indexes.
![test](https://user-images.githubusercontent.com/5678012/101009828-07da2e00-355a-11eb-903f-d03eeb87340c.png)

Reply:
Do you want me to change the type of `sort` in the icelandic index?

Reply:
Would changing the type of the `sort` field in the english index fix this issues.
We need the `icu_collation_keyword` in icelandic to sort icelandic letters correctly

Reply:
I just tried it and it did not work, the type became unknown and I was not able to use it in a visualization